company-box-icons.el

```diff
@@ -248,17 +248,25 @@ specification.md#completion-request-leftwards_arrow_with_hook.")
 	    ((eq annotation "") 'Variable)
 	    (t 'Function)))))
 
+(defconst company-box-icons--anaconda-alist
+  '(("function" . Function)
+    ("statement" . Variable)
+    ("module" . Module)
+    ("keyword" . Keyword)
+    ("class" . Class)
+    ("instance" . Property))
+  "List of icon types to use with Anaconda candidates.")
+
 (defun company-box-icons--anaconda (candidate)
   "Return the icon type that corresponds to CANDIDATE with company-anaconda."
   (when (and (derived-mode-p 'python-mode)
 	     (require 'company-anaconda nil t))
     (--when-let (aref (get-text-property 0 'struct candidate) 1)
-      (cond ((string= it "function") 'Function)
-	    ((string= it "statement") 'Variable)
-	    ((string= it "module") 'Module)
-	    ((string= it "keyword") 'Keyword)
-	    ((string= it "class") 'Class)
-	    ((string= it "instance") 'Property)))))
+      (alist-get it company-box-icons--anaconda-alist nil nil 'string=))))
 
+(declare-function company-tern-property-p "ext:company-tern.el")
```

Comment:
Will these work without `company-tern.el` available?

Reply:
Yes, those are for the compiler so it doesn't complain about undefined functions. It just says to the compiler that those functions exist externally